src/app/component/layout/components/search-popup/search-popup.component.scss

```diff
@@ -13,6 +13,10 @@ h3 {
   margin-bottom: 15px;
 }
 
+.search-item {
+  box-shadow: 1px 4px 8px rgba(100, 114, 125, 0.18);
+}
+
```

Comment:
Create variable for color

Reply:
I think it's a bad idea because it's rgbA, so it has an Alpha channel, and if I replace it with color, the opacity will vanish, and btw we have code like this in 30 files.